backend/loi-service/src/main/java/org/egov/works/repository/rowmapper/LOIRowMapper.java

```diff
@@ -76,7 +76,7 @@ public List<LetterOfIndent> extractData(ResultSet rs) throws SQLException, DataA
                     .defectLiabilityPeriod(defectLiabilityPeriod)
                     .oicId(UUID.nameUUIDFromBytes(oicId.getBytes(StandardCharsets.UTF_8)))
                     .letterStatus(letterStatus)
-                    .additionalDetails(additionalDetails)
+                    .additionalDetails(additionalDetails.asText())
```

Comment:
Why asText()? Additional details is a json object, not a string.

Reply:
We are doing same way for search estimate.